lib/LedgerSMB/Scripts/recon.pm

```diff
@@ -215,43 +215,31 @@ sub _display_report {
     _set_sort_options($recon, $request);
 
     $recon->get;
-    $recon->unapproved_checks;
-
-    my $neg_factor = ($recon->{account_info}->{category} =~ /^[AE]/) ? -1 : 1;
-    $recon->{their_total} *= $neg_factor;
-
-    if ($recon->{account_info}->{category} eq 'A') {
-        $recon->{reverse} = $request->setting->get('reverse_bank_recs');
-    }
-
     _process_upload($recon, $request) unless $recon->{submitted};
-
     $recon->build_totals;
     $recon->build_statement_gl_calc;
     $recon->build_variance;
-    _highlight_suspect_rows($recon);
+
+    if ($recon->{account_info}->{category} eq 'A') {
+        $recon->{reverse} = $request->setting->get('reverse_bank_recs');
+    }
 
     $recon->{submit_enabled} = ($recon->{variance} == 0);
+    _highlight_suspect_rows($recon);
 
     for my $amt_name (qw/ mismatch_our_ mismatch_their_ total_cleared_ total_uncleared_ /) {
         for my $bal_type (qw/ credits debits/) {
             $recon->{"$amt_name$bal_type"} = $recon->{"$amt_name$bal_type"}->to_output(money=>1);
         }
     }
-    $recon->{their_total} *= $neg_factor;
 
     for my $line (@{$recon->{report_lines}}){
         for my $element (qw/ our_balance our_credits our_debits their_balance their_credits their_debits /) {
             $line->{$element} = $line->{$element}->to_output(money => 1);
         }
     }
 
-    for my $field (qw/ cleared_total outstanding_total statement_gl_calc their_total variance /) {
-        $recon->{$field} = $recon->{$field}->to_output(money => 1);
-    }
-
-    for my $field (qw/ our_total beginning_balance /) {
-        $recon->{$field} ||= LedgerSMB::PGNumber->from_db(0);
+    for my $field (qw/ cleared_total outstanding_total statement_gl_calc their_total variance our_total beginning_balance /) {
```

Comment:
It's not the end of the world, but it'd be nice (for code that's not explicitly deprecated) to try and stick to 80-character width. If nothing else, many brail readers have that width, even today. (The something else would be: I've always understood that the 80-character width fits better with the human brain...)